apps/agent/app/src/components/useDownloadData.js

```diff
@@ -26,12 +26,11 @@ async function getDownloadData({ transactions, tokenDetails, resolveAddress }) {
         tokenTransfers.map(async ({ amount, from, to, token }) => {
```

Comment:
To be safe, let's stringify each of the values we're returning in this `map()` (similar to https://github.com/aragon/aragon-apps/pull/1121/files#diff-c6b357d815fc23c7aea312476aabe3beR49).